engine/action_destination_test.go

```diff
@@ -320,13 +322,12 @@ func (ts *EmbeddedDBTestSuite) TestClient_GetDestinations() {
 			require.NoError(t, err)
 			require.NotNil(t, destination)
 
-			trueValue := true
-
-			conditions := models.DestinationFilters{IncludeDeleted: &trueValue}
+			// deleted items should be present by default (empty conditions)
```

Comment:
Thought we agreed to add `IncludeDeleted` flag if we want to include deleted records

Reply:
It's wrong comment - actually it works the opposite.